conf/lookup/BCD-Port.csv

```diff
@@ -11461,7 +11460,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 32107,UK,2,CEG,,Hawarden Airport - CEG,Hawarden Airport - CEG,,1,NULL,1,
 32261,UK,2,ILY,,Islay Airport - ILY,Islay Airport - ILY,,1,NULL,1,
 32265,UK,2,SKY,,Isle Of Skye Airport - SKY,Isle Of Skye Airport - SKY,,1,NULL,1,
-32289,UK,2,MSE,,Kent International Airport - MSE,Kent International Airport - MSE,,1,NULL,1,
+32289,UK,2,MSE,Manston,Kent International Airport - MSE,Kent International Airport - MSE,,1,NULL,1,
 32343,UK,2,KOI,,Kirkwall Airport - KOI,Kirkwall Airport - KOI,,1,NULL,1,
 32346,UK,2,KPT,,KnAirporton - KPT,KnAirporton - KPT,,1,NULL,1,
 32373,UK,2,QLA,,Lasham Airport - QLA,Lasham Airport - QLA,,1,NULL,1,

```

Comment:
Remove, this is duplicated here with correct code:

UK,2,LAS,,Lasham Airport - LAS,Lasham Airport - LAS,,1,NULL,0,